tests/modelci/test_eval_function.py

```diff
@@ -1,6 +1,3 @@
-from nucleus.modelci.eval_function import EvalFunction
-
-
 def test_list_eval_functions(CLIENT):
-    eval_functions = CLIENT.modelci.list_eval_functions()
-    assert all(isinstance(eval_fn, EvalFunction) for eval_fn in eval_functions)
+    eval_functions = CLIENT.modelci.eval_functions
+    print(eval_functions.iou > 0.5)
```

Comment:
Any way we could turn this into an assertion?

Reply:
Changed it to check for right instance type.